spec/spec_helper.rb

```diff
@@ -26,6 +26,15 @@ module ClassMethods
     # runs a test once with real mercury and once with Mercury::Fake
     def itt(name, &block)
       it(name, &block)
+      context 'without publisher confirms' do
+        before :each do
+          real_open = Mercury.method(:open)
+          allow(Mercury).to receive(:open) do |**kws, &k|
+            real_open.call(**kws.merge(wait_for_publisher_confirms: true), &k)
```

Comment:
Why is this passing `wait_for_publisher_confirms: true` if the context is `'without publisher confirms'`?